actions/patch_node_handler.go

```diff
@@ -64,12 +65,17 @@ func (h *patchNodeHandler) Handle(ctx context.Context, action *castai.ClusterAct
 		return err
 	}
 
-	log.Infof("patching node, labels=%v, taints=%v, annotations=%v", req.Labels, req.Taints, req.Annotations)
+	unschedulable := "<nil>"
```

Comment:
This looks strange. %v can handle nil

Reply:
%v handles nil when unschedulable is not provided. When it is provided then %v prints pointer value. It does not dereference when value is provided.